specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Attached to this top line I'll enter a number of comments on the data model proper, to keep them all together.  Sorry I didn't get to them earlier.
Line numbers refer to commit 5abd84e or 140519f